### PR TITLE
rework FAQ

### PR DESCRIPTION
**Pull request by pivilartisant**
Nothing changes in FAQ

Screenshot 2023-08-23 at 17 15 59

Added modularity
Screenshot 2023-08-23 at 17 16 43

Screenshot 2023-08-23 at 17 16 24

### CODE REVIEW

File: src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -38,29 +41,42 @@ export function FAQ(props: IFAQ) {
...
+  const baseIconOpen = <FiPlus size={20} />;
+
+  const hasParentElemement = isChild !== false;
+
   return (
-    <div>
-      <hr
-        className="h-0.5 border-t-0 bg-transparent bg-gradient-to-r 
+    <div className="text-neutral">
+      {hasParentElemement ? (
```
Comment (pivilartisant) [inline src/components/Accordion/Accordion.tsx:56]: if category, has parent element render hr tag, else render empty div

Comment (pivilartisant) [reply src/components/Accordion/Accordion.tsx:56]: I think this is kind of hacky @mazmassa

File: src/components/Accordion/Accordion.tsx [hunk 2]
```diff
@@ -38,29 +41,42 @@ export function FAQ(props: IFAQ) {
...
+
+  const isNotChild = isChild === false;
+
   return (
-    <div>
-      <hr
-        className="h-0.5 border-t-0 bg-transparent bg-gradient-to-r 
+    <div className="text-neutral">
+      {isNotChild ? (
+        <></>
```
Comment (mazmassa) [inline src/components/Accordion/Accordion.tsx:57]: I think you can make the opposite here just using &&, isn't it!?

```suggestion
      {isChild && 
        <hr className="h-0.5 border-t-0 bg-transparent bg-gradient-to-r 
```

File: src/components/Accordion/Accordion.test.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import { Accordion, AccordionCategory, AccordionContent } from '.';
+
+describe('Components | Toggle', () => {
```
Comment (mazmassa) [inline src/components/Accordion/Accordion.test.tsx:5]: Toggle!? 🤔

Comment (pivilartisant) [reply src/components/Accordion/Accordion.test.tsx:5]: arf

File: src/components/Accordion/Accordion.tsx (hunk 2 above)
Comment (pivilartisant) [reply src/components/Accordion/Accordion.tsx:57]: if i pass directly the boolean isChild as argument of ternary and look at the case where i don't pass isChild in args it evaluates to undefined and always equals false

Comment (pivilartisant) [reply src/components/Accordion/Accordion.tsx:57]: the objective is to only say when necessary if is element is not child

Comment (pivilartisant) [reply src/components/Accordion/Accordion.tsx:57]: but i guess it would be acceptable to pass the isChild bool param everytime we use it ?

**Review by mazmassa — APPROVED** [review]

**Pull request merged**